Sources/JExtractSwiftLib/FFM/CDeclLowering/FFMSwift2JavaGenerator+FunctionLowering.swift

```diff
@@ -348,6 +348,46 @@ struct CdeclLowering {
     case .composite:
       throw LoweringError.unhandledType(type)
 
+    case .array(let wrapped) where wrapped == knownTypes.uint8:
+      // Lower an array as 'address' raw pointer and 'count' integer
+      let cdeclParameters = [
+        SwiftParameter(
+          convention: .byValue,
+          parameterName: "\(parameterName)_pointer",
```

Comment:
We have naming inconsistency; these should be $ and not _ but seems we got this inconsistency already from the Data work -- lets fix it separately, since this made the PR kinda big